h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -317,7 +317,9 @@ void appendFailedModelParameters(final Key<Model> modelKey, final MP params, fin
    * @return list of model keys
    */
   public Key<Model>[] getModelKeys() {
```

Comment:
Small note. I personally try to make the JavaDoc of our project a little bit better. So I'd add an information to the Javadoc it is actually the intention and desired behavior to order the model keys lexically.